deepchem/models/torch_models/ferminet.py

```diff
@@ -164,12 +164,19 @@ def loss(self,
             indicates whether the model is pretraining
         """
         criterion = torch.nn.MSELoss()
-        if pretrain:
+        if pretrain[0]:
             psi_up_mo_torch = torch.from_numpy(psi_up_mo).unsqueeze(1)
             psi_down_mo_torch = torch.from_numpy(psi_down_mo).unsqueeze(1)
-            self.running_diff = self.running_diff + criterion(
-                self.psi_up, psi_up_mo_torch.float()) + criterion(
-                    self.psi_down, psi_down_mo_torch.float())
+            self.running_diff = self.running_diff.float() + criterion(
```

Comment:
making it float type as psi)up_mo_torch is double and gives errors when run.
Also added loss for training in the else part.